R/connect.R

```diff
@@ -830,12 +830,12 @@ Connect <- R6::R6Class(
     #' @param name The package name to filter by.
     #' @param page_number The page number.
     #' @param page_size Page size, max 500.
-    packages = function(name = NULL, page_number = 1, page_size = 500) {
+    packages = function(name = NULL, page_number = 1) {
       url <- v1_url("packages")
       query_params <- list(
         name = name,
         page_number = page_number,
-        page_size = page_size
+        page_size = "1000000"
```

Comment:
I arbitrarily picked the 1M value in Python since `requests` has no response body limit. Do you think the same value makes sense here?

IIRC, dogfood is somewhere in the 100k range. Since we haven't been able to test beyond that number, should we use 100k instead? That'll result in a few round trips on dogfood, but at least we will know things don't blow up.

Reply:
Yeah, 100k or 500k seems reasonable.

Reply:
fwiw, I don't _think_ `httr` has a body response limit either